src/components/ExpandableListItem.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import {ListItem, ListItemText, ListItemIcon, Collapse, List} from '@material-ui/core';
+import { ExpandMore, ExpandLess } from '@material-ui/icons';
+
+import { Field } from 'health-level-seven-parser'
+
+const ExpandableListItem: React.FC<{
+  field: Field;
+  expandableKey: string;
+  expandableClassName: string;
+  expandableOnClick: () => void;
+  nestedClassName: string
+}> = (props) => {
+
+  const [open, setOpen] = React.useState(false);
```

Comment:
I prefer to `import React, { useState } from 'react';` up top and just use `useState(false);` down here.